index.html

```diff
@@ -95,7 +107,49 @@
             const icon = new L.Icon.Default();
             icon.options.className = user.contributor ? "contributor" : "";
 
-            L.marker(user.coordinates, { icon }).addTo(map).bindPopup(container);
+            L.marker(user.coordinates, { icon })
+              .addTo(map)
+              .bindPopup(() => {
+                (() => {
```

Comment:
does this need to be an IIFE?

Reply:
Did some lazy refactor and didnt notice I'm already in a nested scope :^)